appveyor.yml

```diff
@@ -0,0 +1,49 @@
+version: 1.0.0_b{build}-{branch}
+
+os: Visual Studio 2015
+
+environment:
+  TERM: dumb
+  COVERALLS_REPO_TOKEN:
+    secure: 5nuCg+faxFPeppoNNcSwVobswAVFUf8ut83vw8CX/4W2y0kZkGmwEfCUxSQWiQDU
+  CI_NAME: appveyor
+  CI_JOB_ID: $(APPVEYOR_JOB_ID)
+  GCOV_PREFIX: $(APPVEYOR_BUILD_FOLDER)
+  matrix:
+  - COMPILER: msvc
+    GENERATOR: "Visual Studio 14 2015"
+    CFLAGS: /MP
+    CXXFLAGS: /MP
+  - COMPILER: mingw
+    GENERATOR: "MSYS Makefiles"
+
+platform:
+  - x86
+  - x64
+
+configuration:
+  - Debug
+  - Release
```

Comment:
I think that for faster overall builds I'm no longer going to build the Release config, just RelWithDebInfo & Debug. Usually the Release build does not provide any more insights than RelWithDebInfo.